production/helm/loki/values.yaml

```
@@ -3204,6 +3204,12 @@ memcached:
     fsGroup: 11211
   # -- The name of the PriorityClass for memcached pods
   priorityClassName: null
+  # -- Readiness probe for memcached pods (probe port defaults to container port)
+  readinessProbe:
```

```suggestion
  readinessProbe:
    tcpSocket:
      port: client
```

Should be worked as well?